CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 * `DissimilarityMatrix` now allows non-hollow matrices. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
 * `DistanceMatrix.from_iterable` now accepts a `validate=True` parameter. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
 
+* Added `IntervalMetadata` object in `skbio.metadata._interval`. [#1356](https://github.com/biocore/scikit-bio/issues/1356)
```

Comment:
- remove newline above this line
- `skbio.metadata._interval` -> `skbio.metadata`
- add mention of `Interval` since it is a public class
- add parens around the issue number (i.e. format this entry like the rest of the changelog)